### ISSUE TITLE
Country Search

### ISSUE THREAD

**Issue by GMoon777** [goveo/react-international-phone#134]
Hi, I love your component, thank you for sharing it.

I was wondering if you had cosidered adding a country search bar to the top of the dropdown? this would make things much easier for the end user and make this the best library out there.

Thanks! 👍

**Comment by Quared** [comment]
Bump

**Comment by tian-yi** [comment]
Yeah would be nice to have this. blinedly typing is kinda hard. eg. when serach United kingdom or United States, UAE is always first one to pop up. 

And agreed what @GMoon777 said., I've tested out a few phone input component, this is indeed the best one.

**Comment by goveo** [comment]
Sorry, don't have much time to implement some features right now.
But I will consider implementing country search input soon as a top-priority feature (it was requested many times...)
